app/observers/registry_observer.rb

```diff
@@ -2,6 +2,14 @@ class RegistryObserver < ActiveRecord::Observer
 
   def after_create(registry)
     registry.create_global_namespace!
+
+    # Now it is possible to create the private namespaces
+    # of all the users that signed into portus _before_
+    # the 1st registry was created
+    # TODO: change code once we support multiple registries
+    User.all.each do |user|
+      user.create_personal_namespace!
+    end
```

Comment:
May be you could put it into a single line (through { }) while we don't support multiple registries.